modules/serverless-rn-1-35-0.adoc

```diff
@@ -18,6 +18,16 @@
 * {ServerlessProductName} now uses Knative for Apache Kafka 1.15.
 * The `kn func` CLI plugin now uses `func` 1.16.
 
+* The current download path for the Knative (kn) client on `mirror.openshift.com` is deprecated and will no longer be available. No automatic redirection will occur. If your project or CI/CD pipelines rely on this URL to install the {ServerlessProductName} CLI, you must update your configuration to use the new location.
++
+** Deprecated URL:
+https://mirror.openshift.com/pub/openshift-v4/clients/serverless/
+
+** New URL:
+https://mirror.openshift.com/pub/cgw/serverless/
```

Comment:
That's correct URL, but at the moment it is not created/ setup finished yet. I'd suggest we only mention deprecated URL won't work from next release. And state something like "further migration details will be provided when available".